modules/test-utils/src/browser-automation/browser-test-driver.ts

```diff
@@ -120,12 +127,17 @@ export default class BrowserTestDriver extends BrowserDriver {
             : url;
 
           // eslint-disable-next-line @typescript-eslint/no-floating-promises
-          this.openPage({
-            url: pageUrl,
+          const page = await this.openPage({
             exposeFunctions,
             onConsole: event => this._onConsole(event),
             onError: reject
           });
+
+          if (config.onStart) {
```

Comment:
Nit: Maybe set defaults to empty function or use `?.()` so we can avoid if statements here?